content/docs/reference/usage/subcommands.md

```diff
@@ -38,6 +38,32 @@ Run the program, either directly on the host or in an emulated environment (depe
 ### flash
 Flash the program to a microcontroller.
 
+The `-monitor` flag can be given to the `flash` command to start the serial
+monitor (see below) immediately after flashing. However, some microcontrollers
+need a split second or two to configure its serial port after flashing, and
+using the `-monitor` flag can fail because the serial monitor starts too
+quickly. In that case, use the `tinygo monitor` command explicitly.
+
+### monitor
+Start the serial monitor program on the serial port that is connected to the
+microcontroller.
```

Comment:
This seems to suggest the monitor is a separate program, but it's built into TinyGo itself.
I'd just say this:

```suggestion
Start the serial monitor on the serial port that is connected to the
microcontroller.
```

Reply:
Done.